File: Java/src/Find Minimum in Rotated Sorted Array.java
public class Solution {
    public int findMin(int[] num) {
        int low = 0;
        int high = num.length - 1;
        while (low < high) {
            int mid = (low + high) /2;
            if (num[mid]>num[high]){
                low = mid + 1;
            }
            else{
                high = mid;
            }
        }
        
        return num[low];
    }
}

File: Python/Find Minimum in Rotated Sorted Array.py
class Solution:
    # @param num, a list of integer
    # @return an integer
    def findMin(self, num):
        low=0
        high=len(num)-1

        while low < high :
            mid=(low + high)//2;
            if num[mid] > num[high] : low = mid + 1
            else: high = mid
        
        return num[low]